src/commands/deploy/latest.js

```diff
@@ -590,7 +591,12 @@ export default async function main(
         }
       }
     } else {
-      const deploymentResponse = await now.fetch(deploymentUrl);
+      const deploymentResponse = await getDeploymentByUrl(now, deploymentUrl);
```

Comment:
The `deploymentUrl` will either be a `now.sh` domain or a custom suffix domain, both have a wildcard cert so this would never really error, would it?

Except if the cert for the custom suffix is missing 🤔

Reply:
@AndyBitz I got an error here in my testing when running `now deploy --target production` (HTTP pretest failure).

Reply:
@mglagola this is weird, because it should be an invalid cert error when accessing the deployment directly, right?

Reply:
@mglagola can you try to curl the `deploymentUrl` and see what you get?

Reply:
@AndyBitz curl returns:

```json
{  
   "error":{  
      "message":"We could not create a certificate for external-test.xyz because HTTP pretest failed",
      "code":"configuration_error",
      "domain":"external-test.xyz",
      "external":false,
      "reason":"Invalid challenge response for external-test.xyz. Make sure the DNS records are correct.",
      "statusCode":449,
      "helpUrl":"https://err.sh/now-cli/cant-solve-challenge",
      "cns":[  
         "external-test.xyz"
      ]
   }
}
```

Reply:
@mglagola what was the url you curled?